basemaps/localization.md

```diff
@@ -66,133 +116,97 @@ Extending our Switzerland example with exonym and endonym from other languages:
 - `name:en` = `Switzerland`
 - `name:es` = `Switzerland`
 - `name:pt` = `Suíça`
-- `name:zh` = `瑞士`
 - `name:zh-Hans` = `瑞士`
 - `name:zh-Hant` = `瑞士`
 - _... many other localized values..._
 
-_NOTE: The Chinese (`zh`) examples above demonstrates how a single language can have multiple writing systems, in this case both simplified Chinese (`zh-Hans`) used in mainland China and tranditional Chinese (`zh-Hant`) used in Taiwan. The value stored in `zh` could be either of those._
-
-## Script of default `name` value
-
-The default (or primary) `name` does not self describe the writing system "script" or character set (alphabetic, stroke-based, or otherwise) used to render the value. When combining with localized `name:*` values. This complicates preferring to "fallback" to another language in the same script family before falling back to characters using a different writing system the reader may not be able to make sense of.
-
-To help solve this, Protomaps characterizes the scipt used in the default `name` value by adding a `pmap:script` tag.
-
-Values in `pmap:script` follow the [ISO 15924](https://unicode.org/iso15924/iso15924-codes.html) standard codes for the representation of names of scripts and are summarized in the table below.
-
-_NOTE: Some languages can be written in more than one script, e.g., Malay can be written in Latin, Arabic, and Thai._
-
-## Common languages, their codes, and scripts
-
-This table summarizes 26 common langauges, their ISO codes, and writing system scripts.
-
-| Language |  Native name | `name:*` property | [ISO 639-2 code](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) | [ISO_639-1 code](https://en.wikipedia.org/wiki/ISO_639-1) | [ISO_15924 script(s)](https://unicode.org/iso15924/iso15924-codes.html) |
-|--------|-----------------|-----------|-----|----|----|
-| Arabic | اَلْعَرَبِيَّةُ | `name:ar` | ara | ar | `Arabic` |
-| Bengali | বাংলা | `name:bn` | ben | bn | `Bengali` |
```

Comment:
What's our longer term plan to support Bengali and Farsi (I think the only ones dropped from this list)?

Reply:
**Bengali** to be added in the future as a positioned glyph font

**Farsi** we call it "Persian"